doc/content/reference/telemetry/identity_server.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Identity Server Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the IdentityServer"
+aliases: "/telemetry/identity-server"
+---
+
+[The Things Stack]({{<ref "/the-things-stack">}})'s Identity Server collects telemetry data, which is used to help us understand how users interact with the product. For example, the current data helps to have a better insight on the user's usage of organizations. The objective with the introduction of telemetry is to be able to better understand what is used so that we can expand and improve on the experience that we provide to our users.
+
```

Comment:
Please replace the last line here as well with the contents mentioned in the CLI section.